deployment/keystone/changeset/deploy_registry.go

```diff
@@ -5,21 +5,22 @@ import (
 
 	"github.com/smartcontractkit/chainlink/deployment"
 	kslib "github.com/smartcontractkit/chainlink/deployment/keystone"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
 )
 
-func DeployCapabilityRegistry(env deployment.Environment, config interface{}) (deployment.ChangesetOutput, error) {
+func DeployCapabilityRegistry(env deployment.Environment, config interface{}) (deployment.ChangesetOutput, *capabilities_registry.CapabilitiesRegistry, error) {
```

Comment:
this will break downstream users.

there ought to be a type assert here, but it's missing. please add it
[type assertion
](https://github.com/smartcontractkit/chainlink/blob/d2817c567cfc67c6a532175296d9d96da7d4ad6e/deployment/keystone/changeset/deploy_forwarder.go#L10)

instead of returning the contract, you can load the contract from the returned addressbook
[contract set
](https://github.com/smartcontractkit/chainlink/blob/d2817c567cfc67c6a532175296d9d96da7d4ad6e/deployment/keystone/state.go#L44)